.dockerignore

```diff
@@ -15,7 +15,10 @@
 .travis.yml
 samples
 test_data
-bin
+bin/
 _dist
 coverage.txt
 workspace/
+my-sessions/
+# dist/
+# assets/swagger/
```

Comment:
Would be good to enable them or delete them

Reply:
done